src/gentropy/dataset/intervals.py

```diff
@@ -74,32 +71,3 @@ def from_source(
         source_class = source_to_class[source_name]
         data = source_class.read(spark, source_path)  # type: ignore
         return source_class.parse(data, gene_index, lift)  # type: ignore
```

Comment:
Just wondering, what is the plan with the interval dataclass?  Do we plan to have a separate DAG to process these and save them somewhere?